### DISCUSSION TITLE
An overall galleries index view

### DISCUSSION THREAD

**Discussion by dancingdog** [ente-io/ente#4466] [Enhancements]
I'm using the Windows and Linux desktop apps on different machines.

I have now added many albums, and find it tedious scrolling back and forth along the gallery thumbnails. I initially though these coulkd be larger but realised this would take away from the space for photos.

How about a Galleries index listing  view, with larger thumbnails of each (and ability to set your own image for thumbnail?)

**Comment by mnvr** [comment]
There is an all albums view that can be invoked using this expansion button at the end of albums listing. 

<img width="233" alt="Screenshot 2024-12-23 at 11 16 23" src="https://github.com/user-attachments/assets/8cf4acf9-e1ce-409a-a000-b973f3e433df" />
<img width="446" alt="Screenshot 2024-12-23 at 11 18 22" src="https://github.com/user-attachments/assets/76e8cb7f-a39f-4fcf-9658-62e40b07dcf8" />

---

(ps. not to take away from the other parts of the feature request, just mentioning in case you were unaware of this listing and it fulfills some of your needs for now)
